runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesOverlay.java

```diff
@@ -108,6 +110,22 @@ private void renderProgressCircle(Graphics2D graphics, XpGlobe skillToDraw, int
 
 		Ellipse2D backgroundCircle = drawEllipse(graphics, x, y);
 
+		drawSkillImage(graphics, skillToDraw, x, y);
+
+		Point mouse = client.getMouseCanvasPosition();
+		int mouseX = mouse.getX() - bounds.x;
+		int mouseY = mouse.getY() - bounds.y;
+
+		// If mouse is hovering the globe
+		boolean hovering = backgroundCircle.contains(mouseX, mouseY);
+
+		if (hovering)
```

Comment:
Couldn't this be moved into the other if statement below?

Reply:
Nah, different rendering orders

Reply:
Actually, fixed.